src/rust/cryptography-x509/src/common.rs

```diff
@@ -57,6 +57,17 @@ pub enum AlgorithmParameters<'a> {
     #[defined_by(oid::RSA_OID)]
     Rsa(Option<asn1::Null>),
 
+    // Used only in PKCS#7 AlgorithmIdentifiers
+    // https://datatracker.ietf.org/doc/html/rfc3565#section-4.1
+    //
+    // From RFC 3565 section 4.1:
+    // The AlgorithmIdentifier parameters field MUST be present, and the
+    // parameters field MUST contain a AES-IV:
+    //
+    // AES-IV ::= OCTET STRING (SIZE(16))
+    #[defined_by(oid::AES_128_CBC_OID)]
+    AesCbc(&'a [u8]),
```

Comment:
Out of curiosity: does `rust-asn1` know to turn this `&[u8]` into the inner value of the `OCTET STRING`, or does this end up containing the raw TLV for the `OCTET STRING`? I suspect it's the former, but we should confirm.

Reply:
`&[u8]` in rust-asn1 are OCTET STRINGs, not raw TLVs.